index.js

```diff
@@ -356,6 +356,22 @@
     ($.Undefined  = type0('Undefined'))
   ];
 
+  //  Enum :: [Any] -> Type
+  $.Enum = function(constants) {
+    var reprs = map(constants, show);
+    return {
+      type: 'sanctuary-def/Enum',
```

Comment:
Yeah, I wasn't sure about that. Should I move the definition `Enum` up to where those types are? I only had it where it was because I was relying on that `$.Array` test.